### PR TITLE
Fix tox environments

### PR DESCRIPTION
**Pull request by pederhan**
Fixes the `coverage` and `lint` tox environments.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -34,6 +34,7 @@ basepython =
     python310: python3.10
     python311: python3.11
     python312: python3.12
+    python3.11
```
Comment (terjekv) [inline tox.ini:37]: Only 3.11? This seems very weird.

**Review by terjekv — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (pederhan) [reply tox.ini:37]: It's the fallback version for when no other version is found for the environment. Uv downloads and installs it. Do you want to run linting and coverage with 3 different interpreters each time?

How did it behave prior to the uv PR?

Comment (terjekv) [reply tox.ini:37]: I *think* coverage ran within the current env, not something by tox?

Comment (pederhan) [reply tox.ini]: I have changed it to just `python3` now, which should select any Python 3 dist (system or uv-managed) that is supported by the project.

**Pull request merged**